src/components/MessagesList/MessagesGroup/Message/MessagePart/Reactions.spec.js

```diff
@@ -282,6 +282,7 @@ describe('Reactions.vue', () => {
 		test('fetches reactions details when they are not available', async () => {
 			// Arrange
 			reactionsStore.resetReactions(token, messageId)
+			console.debug = jest.fn()
```

Comment:
Not needed here as the response is mocked and does not produce an error. Same for others ?

Reply:
Same. It's logging in production, but we don't need it in the jest output